en-US/Players_Guide/themes/pdf-theme.yml

```diff
@@ -0,0 +1,46 @@
+extends:
+  - default
+base:
+  font-size: 12
+page:
+  size: Letter
+title-page:
```

Comment:
This is what the title page can look like given these settings:
<img width="685" height="884" alt="image" src="https://github.com/user-attachments/assets/c4df3198-4796-4f76-a2ea-f1fbb4b8c946" />
Note that the writing is lower on the page than with the other two covers. This is due to how it looks when combined with the cover image (should we want to insert that).